### ISSUE TITLE
Craft 5

### ISSUE THREAD

**Issue by samuel-clara** [codemonauts/craft-instagram-feed#72]
Just checking in if an update for Craft 5 in the works? Thanks 😊

**Comment by kringkaste** [comment]
Yes it is. We update all our plugins in the next days. Sorry for the delay.

**Comment by Alex-Waite** [comment]
Just checking if there is a timeline on when this one might be ready on craft 5? Looking to upgrade a few sites to 5.x over the next week and I like the ease of this plugin 🙂

**Comment by simajkeee** [comment]
any info on craft 5 plugin update?

**Comment by joepagan** [comment]
Hey @kringkaste just curious if you have any plans on a release date for a version which supports craft 5? Thanks

**Comment by vandres** [comment]
@kringkaste Have the plans to support Craft 5 been abandened?

**Comment by samuel-clara** [comment]
Please add Craft 5 support! your plugin is so much simpler and better then alternatives. I would be more than happy to pay for this plugin.

**Comment by wolispace** [comment]
Yet another check in to see how the upgrade to Craft 5 is going or has it been abandoned?

**Comment by duikb00t** [comment]
How is the update going for Craft5? Thanks!

**Comment by samuel-clara** [comment]
is there any news on this? 🙏
